packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.controller.js

```diff
@@ -62,40 +92,20 @@ export default class PciTrainingJobsSubmitController {
       });
   }
 
-  onAddVolume(form) {
-    const volume = {
-      region: form.container.$viewValue.region,
-      container: form.container.$viewValue.name,
-      mountPath: form.mountPath.$viewValue,
-      permission: form.permission.$viewValue,
-    };
-
-    this.job.volumes.push(volume);
-    this.filterContainers();
-  }
-
-  // Dirty hack to validate the oui-inline-adder form
-  // eslint-disable-next-line class-methods-use-this
-  forceSubmitContainer(form) {
-    form.$$controls.forEach((innerForm) => {
-      // eslint-disable-next-line no-param-reassign
-      innerForm.$error = {};
-      innerForm.$setPristine();
+  validateVolume() {
+    this.job.validVolume = true;
+    map(this.httpHeader, (volume) => {
```

Comment:
It is weird to use a `map` here as you don't use its result.
Why not use something like `this.job.validVolume = this.httpHeader.some((volume) => ...)`.

Also, the two `if` conditions below could be merged into one.

Reply:
Have taken care of this